server/src/service.rs

```diff
@@ -7,6 +7,7 @@ use oo7::{
         api::{Properties, SecretInner},
         Algorithm, ServiceError,
     },
+    portal,
```

Comment:
It is only used for item, so just import it as PortalItem or use `oo7::portal::Item`

Reply:
Done, thanks!